docs/framework-fastapi.md

```diff
@@ -297,6 +297,6 @@ Clientele specifically focuses on calling an API and doesn't handle callbacks or
 ## Next Steps
 
 - [Learn about regeneration workflow](usage.md#regenerating)
-- [Configure authentication](examples.md#authentication)
+- [Configure authentication](configuration.md#authentication)
```

Comment:
Hmm. This doesn't actually lead anywhere. I think what we actually want instead is to create a new page called `authentication.md` under the Documentation heading in the mkdocs.yml and a whole page dedicated to explaining the types of authentication we support, with code examples on how to set it using the config object, and what it does under the hood (explain how it sets the headings). We should also mention the `additional_headings` config option on this new page too. Once again this can be short and sweet for now like the testing page and the new exception handling page I suggest in another comment.